types/pdfmake/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-declare-current-package": false
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`, you can try to fix it by moving the code into new files

Reply:
hi @plantain-00 in case I moning the code into 2 files, then there is one problem:
in the official pdfmake's documentation I should use this code:
```
for client-side use:
require ('pdfmake/build/pdfmake.js');
and require ('pdfmake/build/vfs_fonts.js');
```

or:
```
import * as 'pdfmake/build/pdfmake';
import * as 'pdfmake/build/vfs_fonts';
```
but I can't give 'pdfmake/build/pdfmake' name to some file. Whether it is possible to solve this problem?

Reply:
You can try to create a directory named `build` in `pdfmake`, and create files of `pdfmake.d.ts` and `vfs_fonts.d.ts` in it